### ISSUE TITLE
merging chapters/episodes

### ISSUE THREAD

**Issue by jcolomb** [carpentries-incubator/managing-computational-projects#99]
see https://github.com/carpentries-incubator/managing-computational-projects/pull/98, one can call md files in an episode and merge them.

Issue is that the question/objective part does not work (duplicate + at 2 places anyway)

We can move the objective to the Rmd file and keep the questions in the subchapter files.
Then we get 2 boxes with the same objectives, but different questions.

**Comment by jcolomb** [comment]
@zkamvar this works locally, but it seems the github action fails, can you have a look ?

**Comment by zkamvar** [comment]
I found the issue. There were two of them:

1. the child markdown documents were not part of the schedule, so they were explicitly ignored and not copied over to the build directory (which is the feature that allows people to work on episode drafts without affecting build time).
2. the reference to one of the child documents needed to be "Rmd"

I've opened https://github.com/carpentries-incubator/managing-computational-projects/pull/100 that will fix this by merging directly into #98
